src/backend/CassandraBackend.cpp

```diff
@@ -1153,25 +1146,24 @@ CassandraBackend::open(bool readOnly)
         cass_ssl_free(context);
     }
 
-    std::string keyspace = getString("keyspace");
+    auto keyspace = config_.value_or<std::string>("keyspace", "");
```

Comment:
Why not use `value_or_throw`?

Reply:
i was going for the same behaviour as we already had but i agree we could use the throwing version in many places instead